ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -29,7 +31,8 @@ class BugDetailsView extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const { history, from, to, tree, location, bugId, bugzillaData, summary, updateBugDetails } = nextProps;
+    const { history, from, to, tree, location, bugId } = nextProps;
```

Comment:
yeah, I have that on my radar since this app uses those all over the place. Though if I could just remove Redux as part of the "performance improvements" OKR, especially since this is the only app using it.